packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -411,11 +417,19 @@ export abstract class FluidDataStoreContext
 
 	public readonly id: string;
 	private readonly _containerRuntime: IContainerRuntimeBase;
+	/**
+	 * ???
+	 */
 	private readonly parentContext: IFluidParentContextPrivate;
 	public readonly storage: IDocumentStorageService;
 	public readonly scope: FluidObject;
-	// Represents the group to which the data store belongs too.
+	/**
+	 * The loading group to which the data store belongs to.
```

Comment:
@jatgarg To provide more information about about this to make this comment more descriptive.